### DISCUSSION TITLE
Is there any update on multi-language text detection?

### DISCUSSION THREAD

**Discussion by VishalJoshi21** [pemistahl/lingua-py#53] [Q&A]
Hello.
This is my first time starting a discussion and new to GitHub so please bear with me.
I'm facing an issue while detecting multi-language from multi-language text.

**Input**
text1 = 'He turned around and asked: "Entschuldigen Sie, sprechen Sie Deutsch?"'
text2 = '言語確率間の距離は、入力テキストの長さに依存することに注意してください. Be aware that the distance between the language probabilities is dependent on the length of the input text. '

**Output**
text1 = [(Language.ENGLISH, 1.0)]
text2 = [(Language.ENGLISH, 1.0)]

Thank you

**Comment by pemistahl** [comment]
Hi @VishalJoshi21, thanks for your question.

Unfortunately, there is no significant progress on multi-language detection yet due to a lack of free time on my part. But I still plan to start working on this topic during the remainder of this year.
